packages/react-form-renderer/src/tests/form-renderer/__snapshots__/render-form.test.js.snap

```diff
@@ -452,9 +460,11 @@ exports[`renderForm function should render single field from defined componentTy
           hideField={false}
         >
           <Component
+            component="text-field"
             name="foo"
           >
             <div
+              component="text-field"
```

Comment:
again, we have the component on the DOM element.

Reply:
This is actually right. It's just stupid <div {...props} /> component :D 

I caught the prop in all other proper components.